openstack_controller/datadog_checks/openstack_controller/metrics.py

```diff
@@ -634,6 +634,20 @@
     'status': 'status',
 }
 
+HEAT_METRICS_PREFIX = "openstack.heat"
+HEAT_SERVICE_CHECK = f"{HEAT_METRICS_PREFIX}.api.up"
+HEAT_RESPONSE_TIME = f"{HEAT_METRICS_PREFIX}.response_time"
+HEAT_STACK_PREFIX = f"{HEAT_METRICS_PREFIX}.stack"
+HEAT_STACK_COUNT = f"{HEAT_STACK_PREFIX}.count"
+HEAT_STACK_METRICS = {
+    f"{HEAT_STACK_PREFIX}.up": {},
```

Comment:
What do you think about naming this `openstack.heat.stack.completed` since a stack is either completed or not, rather than up/down?

Reply:
Do we know all the possible values for `stack_status`?

Reply:
I don't see much sense in this metric having `stack_status` as a tag

Reply:
It looks like there are more than two statuses like `CREATE_IN_PROGRESS` and `DELETE_IN_PROGRESS` so I think we should keep the tag
 and https://docs.openstack.org/heat/latest/getting_started/create_a_stack.html

Reply:
Sounds good--I've kept the tag.